test/broadcasting.jl

```diff
@@ -133,6 +136,12 @@ end
 end
 
 @testset "normal data frame and data frame row in broadcasted assignment - one column" begin
+    df = DataFrame(a=1, b=2)
+    @test_throws ArgumentError df .= DataFrame(b=1, a=2)
+    @test_throws ArgumentError df .= DataFrame(a=1, c=2)
+    @test_throws ArgumentError df[!, [:a, :b]] .= DataFrame(b=1, a=2)
+    @test_throws ArgumentError df[!, [:a, :b]] .= DataFrame(a=1, c=2)
```

Comment:
This testset seems to be about single columns, right? Better put this elsewhere?

Reply:
also moved